internal/manifest/commit_server_tree.go

```diff
@@ -20,6 +21,7 @@ type OSTreeCommitServer struct {
 	// TODO: should this be configurable?
 	Language string
 
+	platform        platform.Platform
```

Comment:
Do we anticipate that we need the platform outside of `serialise`? It seems like it is another one of these transient properties that should ideally only be relevant during the serialisation process.